sdk/servicebus/service-bus/src/serviceBusMessage.ts

```diff
@@ -1017,6 +1034,7 @@ export class ServiceBusMessageImpl implements ReceivedMessageWithLock {
     this.throwIfMessageCannotBeSettled(receiver, DispositionType.deadletter);
 
     return receiver!.settleMessage(this, DispositionType.deadletter, {
+      propertiesToModify: actualPropertiesToModify,
```

Comment:
This doesn't require the `reason` and `description` params?

Reply:
This particular function uses the 'error' parameter for that info.